example/service/s3/concatenateObjects/README.md

```diff
@@ -0,0 +1,12 @@
+# Example
+
+concatenateObjects is an example using the AWS SDK for Go to concatenate two objects together.
```

Comment:
Also would be great to describe what is going on and how `UploadPartCopy` can be used to concat two objects already in S3 into a new object.